ls1intum/Artemis
# `Development`: Update client dependencies

### Checklist
#### General



### Motivation and Context
Update to the latest dependencies

### Steps for Testing
Try out a quiz creation and make sure changing the duration works correctly with different values

#### Exam Mode Testing
Try out a quiz creation and make sure changing the duration works correctly with different values


### Review Progress


#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage
no changes

### Screenshots
no changes
